docs/migration/5_0.md

```diff
@@ -714,57 +628,7 @@ Due to i18next migration, certain set of keys have been migrated (from `_plural`
 ### OnNavigateFocusService
 
 - This service had been removed in favour of autoscroll fixes made to the `Router`.
-
-### AsmMainUICmmponent
-
-- Added `UserAccountFacade` to constructor.
-- Removed `UserService` from constructor.
-  
-### CustomerEmulationComponent
-
-- Added `UserAccountFacade` to constructor.
-- Removed `UserService` from constructor.
-
-### CxAgentAuthService
-
-- Added `UserProfileFacade` to constructor.
-- Removed `UserService` from constructor.
-
-### SelectiveCartService
-
-- Added `UserProfileFacade` to constructor.
-- Removed `UserService` from constructor.
-
-### SavedCartService
-
-- Added `UserAccountFacade` to constructor.
-- Removed `UserService` from constructor.
-
-### WishListService
-
-- Added `UserAccountFacade` to constructor.
-- Removed `UserService` from constructor.
-
-### UserFormComponent
-
-- Added `UserProfileFacade` to constructor.
-- Removed `UserService` from constructor.
-
-### AdminGuard
-
-- Added `UserAccountFacade` to constructor.
-- Removed `UserService` from constructor. 
-
-### B2BUserEffects 
-
-- Added `UserAccountFacade` to constructor.
-- Removed `UserService` from constructor.
-  
-### ApproverGuard
```

Comment:
B2BUserEffects and ApproverGuard are missing for migrations

Reply:
They were not exposed in the public api in 4.3, so no breaking change material for those classes.